tests/test_hyperbolic_space_coordinates.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
Nice! Also, good idea to put it in a separate test file, we should do this for the rotations too.